pkg/deviceclaimingserver/enddevices/ttjsv2/ttjs.go

```diff
@@ -109,7 +109,7 @@ var (
 	errDeviceNotProvisioned = errors.DefineNotFound("device_not_provisioned", "device with EUI `{dev_eui}` not provisioned") //nolint:lll
 	errDeviceNotClaimed     = errors.DefineNotFound("device_not_claimed", "device with EUI `{dev_eui}` not claimed")
 	errDeviceAccessDenied   = errors.DefineInvalidArgument("device_access_denied", "access to device with `{dev_eui}` denied: device is already claimed or the owner token is invalid") //nolint:lll
-	errUnauthenticated      = errors.DefineInternal("unauthenticated", "unauthenticated")
+	errCredentials          = errors.DefineInternal("unauthenticated", "unauthenticated")
```

Comment:
```suggestion
	errCredentials          = errors.DefineInternal("credentials", "invalid credentials")
```

Reply:
And run `tools/bin/mage go:messages js:translations`

Reply:
Sorry, I forgot to rewrite the message but now I did. Thanks for the suggestion.